packages/jazz-tools/src/tools/coValues/account.ts

```diff
@@ -290,12 +269,6 @@ export class Account extends CoValueBase implements CoValue {
 
       this.profile = Profile.create({ name: creationProps.name }, profileGroup);
       profileGroup.addMember("everyone", "reader");
-    } else if (this.profile && creationProps) {
-      if (this.profile.$jazz.owner[TypeSym] !== "Group") {
-        throw new Error("Profile must be owned by a Group", {
-          cause: `The profile of the account "${this.$jazz.id}" was created with an Account as owner, which is not allowed.`,
-        });
-      }
```

Comment:
I don't think this check is needed anymore since the owner is always a Group, but I'm not sure why it was necessary in the first place. Calling this out just in case (cc @gdorsi)